modules/ob-deleting-the-builds-component-and-custom-resources.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// builds/uninstalling-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id='ob-deleting-the-builds-component-and-custom-resources_{context}']
+= Deleting the {builds-shortname} components and custom resources
+
+Delete the custom resources (CRs) that were created by default during Operator installation.
+
+
```

Comment:
You mention in the assembly that you have to be a cluster admin to perform the procedures. This should also be included as a Prerequisite for the procedures.